tests/unit/test_errors.py

```diff
@@ -0,0 +1,22 @@
+from craft_cli.errors import CraftError
+
+
+def test_crafterror_is_comparable():
```

Comment:
Please add two tests to check for equal and different for the rest of `CraftError` attributes (two tests is enough, each parametrized to test the different attributes).

Thanks!

Reply:
I created two tests - I hope they are the way you intended.